apps/condo/pages/tls.tsx

```diff
@@ -0,0 +1,391 @@
+import enAndroid from '@condo/lang/en/pages/tls/android.md'
```

Comment:
That's a bad approach of using things. You're importing 5 entire pages for each locale, so all of them will be downloaded on client. Use `getServerSideProps` approach to load only needed content

Reply:
Fixed